posthog/models/feature_flag/flag_matching.py

```diff
@@ -489,22 +493,24 @@ def get_all_feature_flags(
         # with a small timeout
         try:
             with execute_with_timeout(FLAG_MATCHING_QUERY_TIMEOUT_MS):
-                if person_id is None:
```

Comment:
Why is this condition removed?

Reply:
because if the merge happened between us finding the `person_id` and trying to add overrides for it, this would then break because this person_id doesn't exist anymore.